packages/browser-wallet/src/popup/pages/Setup/GenerateSeedPhrase.tsx

```diff
@@ -28,7 +28,14 @@ export default function GenerateSeedPhrase() {
             <div className="onboarding-setup__page-with-header">
                 <div className="onboarding-setup__page-with-header__description">{t('recoveryPhrase.description')}</div>
                 <div className="p-10">
-                    <ControlledTextArea value={seedPhrase} readOnly />
+                    <ControlledTextArea
+                        value={seedPhrase}
+                        readOnly
+                        onCopy={(e) => {
```

Comment:
We agreed that it should possible to copy and paste the seed phrase into e.g. a password manager, so you should not block copying here.

Reply:
We did? :thinking:

Reply:
Yes, for the above reason. Personally I think it would be fine to block it, but then people will probably just screenshot it instead, when they should just write it down by hand.